actors/user/src/main/java/org/sunbird/user/actors/TenantMigrationActor.java

```diff
@@ -138,12 +146,58 @@ private void migrateUser(Request request) {
     // send the response
     sender().tell(response, self());
     // save user data to ES
+    notify(userDetails);
```

Comment:
move notify method after saveUserDetailsToEs . make sure if any exception occured in saveUserDetailsToEs can be handle there it self.